lib/mapObjects/CRewardableObject.h

```diff
@@ -271,7 +269,11 @@ class DLL_LINKAGE CRewardableObject : public CArmedInstance
 		h & onVisited;
 		h & onEmpty;
 		h & visitMode;
-		h & soundID;
+		if(version < 778 && !h.saving)
```

Comment:
Remove `&& !h.saving` condition. Initlialize soundID.

Reply:
Yeah I get ``` !h.saving``` likely make no sense, but we do have it in some other code. Should we get rid of it too?

Reply:
Not in this branch